gui/src/app/pages/HomePage/DataGenerationWindow/DataRFileEditor.tsx

```diff
@@ -121,6 +121,21 @@ if (typeof(data) != "list") {
     [fileContent, editedFileContent, handleRun, status, handleHelp],
   );
 
+  const contentOnEmpty = useMemo(() => {
+    const spanElement = document.createElement("span");
+    const t1 = document.createTextNode(
+      "Define a dictionary called data to update the data.json. ",
```

Comment:
```suggestion
      "Define a list called data to update the data.json. ",
```

R nomenclature suggestion

Reply:
Done